### PR TITLE
Special cov custom templates

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @collin @MichaelButkovic

### CODE REVIEW

**Comment by benghaziboy** [comment]
I based the default off of avclub's templating structure.

**Comment by benghaziboy** [comment]
FELLAS

File: bulbs/special_coverage/models.py [hunk 1]
```diff
@@ -60,20 +84,9 @@ def save(self, *args, **kwargs):
...
-                raise ValidationError("""The End Date requires a Start Date value.""")
-            elif self.end_date < self.start_date:
-                raise ValidationError("""The End Date must not precede the Start Date.""")
-        if self.start_date and not self.end_date:
-            raise ValidationError("""The Start Date requires an End Date.""")
-
     def _save_percolator(self):
-        """saves the query field as an elasticsearch percolator
+        """
+        S`aves the query field as an elasticsearch percolator
```
Comment (MichaelButkovic) [inline bulbs/special_coverage/models.py]: 'Saves'

Comment (benghaziboy) [inline bulbs/special_coverage/models.py]: pound sand

File: bulbs/content/models.py [hunk 2]
```diff
@@ -129,11 +129,6 @@ class ContentManager(PolymorphicManager, IndexableManager):
     a specialized version of `djes.models.SearchManager` for `bulbs.content.Content`
     """
 
-    # def __getattr__(self, name):
```
Comment (MichaelButkovic) [inline bulbs/content/models.py]: if you wanna add a TODO or just remove this, that'd be cool

**Comment by MichaelButkovic** [comment]
:+1:

File: bulbs/content/models.py (hunk 2 above)
Comment (benghaziboy) [inline bulbs/content/models.py]: I did just remove it

File: bulbs/content/models.py [hunk 3]
```diff
@@ -129,11 +129,6 @@ class ContentManager(PolymorphicManager, IndexableManager):
...
-    #     if name.startswith('__'):
-    #         return super(PolymorphicManager, self).__getattr__(self, name)
-    #     return getattr(self.all(), name)
-
```
Comment (mparent61) [inline bulbs/content/models.py]: scary code I'm glad is gone

File: bulbs/special_coverage/models.py [hunk 4]
```diff
@@ -37,17 +37,41 @@ def __unicode__(self):
...
+        self.clean_videos()
+
+    def clean_publish_dates(self):
+        """
+        If an end_date value is provided, the start_date must be less.
+        """
+        if self.end_date:
+            if not self.start_date:
+                raise ValidationError("""The End Date requires a Start Date value.""")
+            elif self.end_date < self.start_date:
```
Comment (mparent61) [inline bulbs/special_coverage/models.py]: should this be "<=" based on docstring above?

**Comment by mparent61** [comment]
:+1:

**Comment by collin** [comment]
Looks good re: custom per-slug templates as far as our annual year is concerned.

OAY had some context data it was relying on, but I think we achieve the same ends with templatetags.

**Pull request merged**